packages/dd-trace/src/appsec/reporter.js

```diff
@@ -91,19 +91,20 @@ function reportWafInit (wafVersion, rulesVersion, diagnosticsRules = {}) {
   incrementWafInitMetric(wafVersion, rulesVersion)
 }
 
+function setOrAddTag (rootSpan, metricName, value) {
+  if (value) {
+    value += rootSpan.context()?._tags[metricName] || 0
```

Comment:
this way of reading a tag from a span seems a bit clunky but it is used widely...